specification/network/resource-manager/Microsoft.Network/stable/2023-06-01/examples/BastionShareableLinkDelete.json

```diff
@@ -23,7 +23,7 @@
     "200": {},
```

Comment:
want to add 204 as another example response since it is usually a valid response for ARM delete requests too?

Reply:
Same as my reply for the previous comment. This PR is for fixing mistakes for existing endpoint, not plan for changing endpoint behavior. Thanks!